README.md

```diff
@@ -69,24 +69,49 @@ jobs:
 
 ### Uploading Results to GitHub
 By default, the dotTEST action always generates a SARIF report. When you upload the report to GitHub, the results will be presented as GitHub code scanning alerts. This allows you to review the results of code analysis with Parasoft dotTEST directly on GitHub as part of your project.
-To upload the SARIF report to GitHub, modify your workflow to add the `upload-sarif` action, which is part of the [github/codeql-action](https://github.com/github/codeql-action) repository):
+To upload the SARIF report to GitHub, modify your workflow to add the `upload-sarif` action, which is part of the [github/codeql-action](https://github.com/github/codeql-action) repository).
+
+**Important!** To automatically upload the results, ensure that the path to the SARIF file is configured as the `${{ steps.[dottest_action_id].outputs.report }}` variable. In the following example the id of the dotTEST action is `dottest`.
 
 ```yaml
-    - name: Upload results to GitHub
+# Run Parasoft dotTEST Analysis and generate .sarif report.
+- name: Run dotTEST analyzer
+  id: dottest
+# Optional parameters
```

Comment:
`uses` property is also mandatory. I'm thinking about removing "Optional parameters" comment and leave only the second line

Reply:
I've added the "uses" property and removed "optional parameters".

Reply:
Looks good